src/node/quote_endorsements_client.h

```diff
@@ -120,9 +120,6 @@ namespace ccf
           }
           if (msg->data.request_id >= msg->data.self->last_received_request_id)
           {
-            LOG_FAIL_FMT(
-              "Timed out reaching endorsement server {}", msg->data.endpoint);
```

Comment:
Why removing this log?

Reply:
The first instance happens nearly instantly after the initial fetch attempt, and is misleading. I don't think fail is right anyway, since this is retried repeatedly until timeout. I am going to update the logging on the endorsement fetching separately.